File: src/main/java/my/chat/route/APIRoute.java
package my.chat.route;

import com.jfinal.config.Routes;
import my.chat.controller.APIController;

/**
 * @author lyu lyusantu@gmail.com
 * @version V1.0
 * @Title: ${file_name}
 * @Description: ${todo}
 * @date ${date} ${time}
 */
public class APIRoute extends Routes {

	public void config() {
		setBaseViewPath("/WEB-INF/view");
		add("/api", APIController.class);
	}
}

File: src/main/java/my/chat/route/APIRoute.kt
package my.chat.route

import com.jfinal.config.Routes
import my.chat.controller.APIController

/**
 * @author lyu lyusantu@gmail.com
 * *
 * @version V1.0
 * *
 * @Title: ${file_name}
 * *
 * @Description: ${todo}
 * *
 * @date ${date} ${time}
 */
class APIRoute : Routes() {

    override fun config() {
        baseViewPath = "/WEB-INF/view"
        add("/api", APIController::class.java)
    }
}
